server/api/search.py

```diff
@@ -0,0 +1,73 @@
+
+import json
+from flask import make_response, request, Blueprint
+from models.recipe import Recipe
+from models.profile import Profile
+from .auth import authenticate
+from db import session
+search_views = Blueprint('search', __name__, url_prefix="/search")
+
+@search_views.route("/cuisines/", methods=["GET"])
+@authenticate
+def search_cuisines():
+    """
+    Search for chefs or recipes based on GET parameters
+    Return the proper info exampled in the mockup
+    or give back an error
+    """
+    search_by = request.args.get('search_by')
+    cuisine = request.args.get('cuisine')
+    result = []
+    response_dict = {}
+    # Make sure values were passed
+    if search_by != "recipe" and search_by != "chef":
+        return {
```

Comment:
Can you align your responses to this format `return jsonify({response}), status`?

Reply:
Sure, I'll do that now